src/plugins/bbsync/sync_thread.cpp

```diff
@@ -100,21 +101,34 @@ BlackBoardSynchronizationThread::init()
 	wsl_local_  = new SyncWriterInterfaceListener(this, logger, (peer_ + "/local").c_str());
 	wsl_remote_ = new SyncWriterInterfaceListener(this, logger, (peer_ + "/remote").c_str());
 
+	init_rb_sync();
+
 	if (!check_connection()) {
 		logger->log_warn(name(), "Remote peer not reachable, will keep trying");
 	}
 
-	logger->log_debug(name(), "Checking for remote aliveness every %u ms", check_interval);
-	timewait_ = new TimeWait(clock, check_interval * 1000);
+	logger->log_debug(name(), "Checking for remote aliveness every %u ms", check_interval_);
+	timewait_ = new TimeWait(clock, check_interval_ * 1000);
+}
+
+void
+BlackBoardSynchronizationThread::init_rb_sync()
```

Comment:
what does `rb` stand for? i guess it makes sense to use `hb` or `heartbeat` instead as it is more intuitive. Also `sync` is kinda misleading, the method actually could be named `init_heartbeat_if()`.